js/tutorial.js

```diff
@@ -20,48 +52,133 @@ Tutorial.prototype.constructor = Tutorial;
+                this.game.mouse.y < this.backButtonHitbox.bottom && this.game.mouse.y > this.backButtonHitbox.top) {
+                
+                this.backButtonHover = true;
+            }
         }
+        
+        if (this.game.click) {
+            if (this.game.click.x < this.backButtonHitbox.right && this.game.click.x > this.backButtonHitbox.left &&
+                this.game.click.y < this.backButtonHitbox.bottom && this.game.click.y > this.backButtonHitbox.top) {
+                
+                this.isDone = true;;
+            }
+        }
+    }
+};
 
+Tutorial.prototype.draw = function (ctx) {
+    this.background.draw(ctx);
+    
+    // For a darker background
+    drawRoundedRect(ctx, 0, 0, 800, 600, 0, "rgba(0, 0, 0, 0.3)", "rgb(0, 0, 0)");
+    
+    // Back button
+    ctx.drawImage(ASSET_MANAGER.getAsset("./img/back.png"), 20, 490, 90, 90);
+   
+    // Objective
+    drawTextWithOutline(ctx, "26px Impact", "Objective:", 20, 40, 'indigo', 'white');
+    drawTextWithOutline(ctx, "22px Impact", "Gain points by being the sole goat on the hill!", 170, 40, 'black', 'white');
+    this.hill.draw(ctx);
+    
+    // Controls
+    drawTextWithOutline(ctx, "26px Impact", "Controls:", 20, 100, 'indigo', 'white');
+    drawTextWithOutline(ctx, "22px Impact", "Press Esc to pause the game", 170, 100, 'black', 'white');
+    drawTextWithOutline(ctx, "22px Impact", "Hold jump key to jump higher", 170, 130, 'black', 'white');
+    drawTextWithOutline(ctx, "22px Impact", "Hold attack key to charge up an attack", 170, 160, 'black', 'white');
+    drawRoundedRect(ctx, 525, 147, 70, 10, 2, "rgba(255, 255, 0, .5)", "rgb(255, 0, 0)");       // Charge meter
+    drawRoundedRect(ctx, 525, 147, this.charge, 10, 2, "rgb(0, 255, 0)", "rgb(255, 0, 0)");     // Charge meter
+    
+    // Blue goat animations and controls
+    drawTextWithOutline(ctx, "22px Impact", "Player 1 (Blue goat)", 170, 190, 'blue', 'white');    
+    this.runRightAnimationBlue.drawFrame(this.game.clockTick, ctx, 170, 200, 0.65);
+    drawTextWithOutline(ctx, "22px Impact", "A   D", 180, 300, "blue", "white");
+    drawTextWithOutline(ctx, "22px Impact", "Shift to run", 180, 340, "blue", "white");
+    this.jumpRightAnimationBlue.drawFrame(this.game.clockTick, ctx, 250, 200, 0.65);
+    drawTextWithOutline(ctx, "22px Impact", "W", 280, 300, "blue", "white");
+    this.attackRightAnimationBlue.drawFrame(this.game.clockTick, ctx, 360, 200, 0.65);
+    this.attackAuraRightAnimationBlue.drawFrame(this.game.clockTick, ctx, 360 - 50, 200 - 15, 0.65 * 5);
+    drawTextWithOutline(ctx, "22px Impact", "F", 390, 300, "blue", "white");
+    
+    // Green goat animations and controls
+    drawTextWithOutline(ctx, "22px Impact", "Player 2 (Green goat)", 500, 190, 'green', 'white');
+    this.runRightAnimationGreen.drawFrame(this.game.clockTick, ctx, 500, 200, 0.65);
+    drawTextWithOutline(ctx, "22px Impact", "◀,   ▶", 510, 300, "green", "white");
```

Comment:
The comma after `◀` gets lost. Perhaps consider using keyboard icons found from the internet for all the keys if there's enough room